### PR TITLE
Aligned REST parameters

### PR DESCRIPTION
**Pull request by psy2848048**
* Added `fee` in bonding/unbonding
* Aligned `gas` and `gas_price`

### CODE REVIEW

File: x/executionlayer/client/rest/msgCreator.go [hunk 1]
```diff
@@ -118,7 +119,7 @@ func bondUnbondMsgCreator(bondIsTrue bool, w http.ResponseWriter, cliCtx context
 		bondingUnbondingAbi = grpc.MakeArgsUnBonding(req.Amount)
 	}
 	paymentCode := grpc.LoadWasmFile(os.ExpandEnv("$HOME/.nodef/contracts/standard_payment.wasm"))
-	paymentAbi := grpc.MakeArgsStandardPayment(new(big.Int).SetUint64(req.GasPrice))
+	paymentAbi := grpc.MakeArgsStandardPayment(new(big.Int).SetUint64(req.Fee))
```
Comment (yoosah) [inline x/executionlayer/client/rest/msgCreator.go]: Is this possible? Where is this req.Fee initialized?

Comment (psy2848048) [reply x/executionlayer/client/rest/msgCreator.go]: In bondReq struct:

```golang
type bondReq struct {
	ChainID              string `json:"chain_id"`
	TokenContractAddress string `json:"token_contract_address"`
	PubkeyOrName         string `json:"pubkey_or_name"`
	Amount               uint64 `json:"amount"`
	GasPrice             uint64 `json:"gas_price"`
	Fee                  uint64 `json:"fee"`
	Memo                 string `json:"memo"`
}
```

`Fee` can be collected from JSON request

Comment (psy2848048) [reply x/executionlayer/client/rest/msgCreator.go]: About the previous `GasPrice`, it was mistake. It should have been `Fee`

**Comment by yoosah** [comment]
psy bro, how about to merge this?

**Comment by psy2848048** [comment]
Too much changes after the push.
Close this and re-push it

**Pull request closed without merging**